docs/components/experimental/rpa/camunda-rpa-framework-library.md

```diff
@@ -0,0 +1,101 @@
+---
+id: camunda-rpa-framework-library
+title: Camunda RPA framework library
+description: "Learn how Camunda offers a custom library for the RPA framework, providing out-of-the-box functionality to manipulate Camunda variables within RPA scripts."
+---
+
+Camunda offers a custom library for the **robotic process automation (RPA)** framework, providing out-of-the-box functionality to manipulate [Camunda variables](/components/concepts/variables.md) within RPA scripts.
+
+## Including the Camunda library
+
+To use Camunda-provided keywords, add the Camunda library in the [Camunda RPA runtime documentation](/components/experimental/rpa/rpa-integration-with-camunda.md) to your Robot file:
+
+```powershell
```

Comment:
Looks like we do not have custom syntax highlighting for this code. Let's use the correct language annotation in case this impacts accessibility labels as well:
```suggestion
```robot
```